libbeat/outputs/elasticsearch/client.go

```diff
@@ -38,7 +38,11 @@ import (
 	"github.com/elastic/beats/v7/libbeat/testing"
 )
 
-var errPayloadTooLarge = errors.New("the bulk payload is too large for the server. Consider to adjust `http.max_content_length` parameter in Elasticsearch or `bulk_max_size` in the beat. The batch has been dropped")
+var (
+	errPayloadTooLarge = errors.New("the bulk payload is too large for the server. Consider to adjust `http.max_content_length` parameter in Elasticsearch or `bulk_max_size` in the beat. The batch has been dropped")
+
+	ErrTooOld = errors.New("Elasticsearch is too old. If you would like to connect to older instances set output.elasticsearch.allow_older_versions to true.")
```

Comment:
Are these errors user facing? If they are, could we be more specific? Something like `Elasticsearch version must be greater than or equal to the beat version. If you ...."`. 

We could also wrap this and format the version into the error this so the user action is explicit like `Upgrade Elasticsearch to vX.X.X or newer: Elasticsearch is too old. If you...` or equivalent.